src/test/java/org/jabref/logic/importer/util/GrobidServiceTest.java

```diff
@@ -12,16 +12,17 @@
 import org.jabref.logic.importer.fileformat.PdfGrobidImporterTest;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
 import org.jabref.testutils.category.FetcherTest;
 
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.mockito.Answers;
 
+import static org.junit.Assert.assertTrue;
```

Comment:
You have the wrong junit, it. must be import static org.junit.jupiter.api.Assertions.assertTrue;